linux/opt/sage/bin/make_env_vars_file.sh

```diff
@@ -73,10 +73,9 @@ export EC2_INSTANCE_ID=$EC2_INSTANCE_ID
 export ROOT_DISK_ID=$ROOT_DISK_ID
 export DEPARTMENT=$DEPARTMENT
 export PROJECT=$PROJECT
+export ACCESS_APPROVED_ROLEID=$ACCESS_APPROVED_ROLEID
 export OWNER_EMAIL=$OWNER_EMAIL
 export OIDC_USER_ID=$PRINCIPAL_ID 
 export OIDC_USERNAME=$SYNAPSE_USERNAME
 export PRODUCT_NAME=$PRODUCT_NAME
-export PRODUCT_ACCESS_APPROVED_ROLEID=$PRODUCT_ACCESS_APPROVED_ROLEID
 EOM
-chmod +x "$OUTPUT_FILE"
```

Comment:
why did you remove this?  we need it.

Reply:
I don't think we do... the next step is just `source file.sh`, which doesn't exec it.

Reply:
not all of them.  some of them just do `. instance_env_vars.sh`